admin/src/components/Configuration/index.js

```diff
@@ -83,6 +87,16 @@ const Configuration = () => {
           paymentMethods,
         });
       }
+      // call github api to get the latest version of the plugin
+      const responseGithub = await fetch(
+        'https://api.github.com/repos/manishkatyan/strapi-stripe/releases/latest'
+      );
+      const data = await responseGithub.json();
+
+      // compare the latest version with the current version
+      if (data.tag_name !== pluginPkg.version) {
```

Comment:
Instead of != i think we should use use greater than expression

Reply:
sure